docs/notebooks/2_getting_started.ipynb

```diff
@@ -29,7 +29,9 @@
    "cell_type": "code",
```

Comment:
<p>typo in computes on first line.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/xscen/pull/231/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>